packaging/fedora/snapd.spec

```diff
@@ -138,6 +145,11 @@ Requires:       bash-completion
 Requires:       %{name}-selinux = %{version}-%{release}
 %endif
 
+%if %{with apparmor}
+BuildRequires:  libapparmor-devel
```

Comment:
Why are we doing this up here? This is repeated in the snap-confine section, where it belongs...